### PR TITLE
Cleanup and refactor switch cases

### PR DESCRIPTION
**Pull request by Isabello**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -230,163 +224,104 @@ class AppUnconnected extends Component {
...
         break;
-       // On any string click change to address focus
-      case ADDRESS_CLICK:
-        this.props.adjustTelescopeForAddress(fixedText);
-        break;
 
-      // Add an address to the list from menu
+      // Microscope Clicks
+      case KEYRING_SELECT_CLICK:
+        this.props.aimMicroscope({keyringIndex: fixedText});
```
Comment (rudi-cilibrasi) [inline src/App.jsx]: should use new performAdjustMicroscopeKeyring etc functions here right?

File: src/App.jsx [hunk 2]
```diff
@@ -230,163 +224,104 @@ class AppUnconnected extends Component {
...
+        // TODO: Take selection and aim microscope at keyring
+        break;
+      case IDENTITY_SELECT_CLICK:
+        this.props.aimMicroscope({keyringIndex: this.props.keyring, identityIndex: fixedText});
```
Comment (rudi-cilibrasi) [inline src/App.jsx]: thought we had this functions already now too

File: src/App.jsx [hunk 3]
```diff
@@ -230,163 +224,104 @@ class AppUnconnected extends Component {
...
         break;
 
       case TX_SET_NAME_CLICK:
-        userTxObj = {
+        readyTxObj = {
           txKind: 'SetName',
-          chainId: this.props.chainId,
-          signer: this.props.profileIdentity[0],
-          name: this.state.nameInput,
+          ...userTxObj
```
Comment (rudi-cilibrasi) [inline src/App.jsx]: this is backware the spread should come before the extra info

File: src/App.jsx (hunk 1 above)
Comment (Isabello) [reply src/App.jsx]: next pr

**Pull request merged**